app/controllers/users_controller.rb

```diff
@@ -0,0 +1,6 @@
+class UsersController < ApplicationController
+  def show
```

Comment:
- [ ] Great job adding the show action for users. As demanded by project requirements, kindly add the index action for users. Note; for this milestone, you can leave your actions blank.